docs/detections/api/exceptions/api-create-rule-default-exception-item.asciidoc

```diff
@@ -0,0 +1,393 @@
+[[exceptions-api-create-rule-default-exception-item]]
+=== Create exception item for individual rules
```

Comment:
```suggestion
=== Create exceptions for individual rules
```

Reply:
Addressed in [1c8efcb](https://github.com/elastic/security-docs/pull/4021/commits/1c8efcbc95ef1df4d4ecd15eaf20ebc36290f0e2).